db/migrate/20221024190158_add_index_to_building_name.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIndexToBuildingName < ActiveRecord::Migration[6.1]
```

Comment:
This was the rubocop issue I was getting: https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsuniquevalidationwithoutindex

But since, as you pointed out, it already had an index, I'm going to guess I had something off in my local environment?  I'm not getting that rubocop anymore locally.